tools/bigquery-to-xml/README.md

```diff
@@ -0,0 +1,48 @@
+# BigQuery to XML Conversion
+This simple Python script allows you to take the output of a BigQuery query and convert the output into an XML file. This works best with nested and repeated fields and contains some customizations like custom root nodes.
+
+
+## User Guide
+
+This tool assumes it will be run by an entity with the appropriate permissions to query BigQuery and that the default project has been configured.
+
+Necessary Python packages are included in requirements.txt.
```

Comment:
```suggestion
Necessary Python packages are included in `requirements.txt` file.
```
See: https://developers.google.com/style/filenames#referring-filenames > Referring to filenames